### DISCUSSION TITLE
How to write attribute value in such a way that we can use different value for that attribute in each component call?

### DISCUSSION THREAD

**Discussion by shubhamdevs** [fastn-stack/fastn#1231] [Q&A]
![Fastn Attribute Doubt](https://github.com/fastn-stack/fastn/assets/96139901/2e5273e6-1e8a-4a8a-8a91-39fb359722f7)

Here I want to use different values of `min-width.fixed.px` in each component call. How to write the attribute in the component?

**Comment by amitu** [comment]
Hey @shubhamdevs, I am not sure I understand your question. You can simply pass different values to different components:

```ftd
-- ftd.text:
min-width.fixed.px: 10

-- ftd.text:
min-width.fixed.px: 20
```

If you want to pass same value to different components you can use a variable:

```ftd
-- integer min-width: 10

-- ftd.text:
min-width.fixed.px: $min-width

-- ftd.text:
min-width.fixed.px: $min-width
```
